Sources/Public/CalendarView.swift

```diff
@@ -688,10 +691,6 @@ extension CalendarView: UIScrollViewDelegate {
 
     let isUserInitiatedScrolling = scrollView.isDragging && scrollView.isTracking
 
-    if isUserInitiatedScrolling, scrollToItemDisplayLink != nil {
-      scrollToItemDisplayLink?.invalidate()
```

Comment:
`scrollToItemContext` is set to `nil` just a few lines down, so no need to manually invalidate here anymore.